<ui version="4.0" stdsetdef="1" >
  <author>Sebastian Trueg</author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>base_K3bAudioTrackWidget</class>
  <widget class="QWidget" name="K3bAudioTrackWidget" >
    <property name="windowTitle" >
      <string>K3bAudioTrackWidget</string>
    </property>
    <layout class="QHBoxLayout" >
      <property name="margin" >
        <number>0</number>
      </property>
      <item>
        <widget class="QTabWidget" name="m_mainTab" >
          <widget class="QWidget" name="tab" >
            <attribute name="title" >
              <string>C&amp;D-Text</string>
            </attribute>
            <layout class="QVBoxLayout" >
              <item>
                <layout class="QGridLayout" >
                  <item row="4" column="1" >
                    <widget class="KLineEdit" name="m_editSongwriter" />
                  </item>
                  <item row="4" column="0" >
                    <widget class="QLabel" name="textLabel4" >
                      <property name="text" >
                        <string>So&amp;ngwriter:</string>
                      </property>
                      <property name="buddy" stdset="0" >
                        <cstring>m_editSongwriter</cstring>
                      </property>
                      <property name="wordWrap" >
                        <bool>false</bool>
                      </property>
                    </widget>
                  </item>
                  <item row="3" column="1" >
                    <widget class="KLineEdit" name="m_editArranger" />
                  </item>
                  <item row="3" column="3" >
                    <widget class="KLineEdit" name="m_editComposer" />
                  </item>
                  <item rowspan="1" row="5" column="1" colspan="3" >
                    <widget class="KLineEdit" name="m_editMessage" />
                  </item>
                  <item rowspan="1" row="1" column="0" colspan="4" >
                    <widget class="Line" name="line2" >
                      <property name="frameShape" >
                        <enum>QFrame::HLine</enum>
                      </property>
                      <property name="frameShadow" >
                        <enum>QFrame::Sunken</enum>
                      </property>
                    </widget>
                  </item>
                  <item rowspan="2" row="2" column="0" colspan="1" >
                    <widget class="QLabel" name="textLabel3" >
                      <property name="text" >
                        <string>&amp;Arranger:</string>
                      </property>
                      <property name="buddy" stdset="0" >
                        <cstring>m_editArranger</cstring>
                      </property>
                      <property name="wordWrap" >
                        <bool>false</bool>
                      </property>
                    </widget>
                  </item>
                  <item rowspan="2" row="2" column="2" colspan="1" >
                    <widget class="QLabel" name="textLabel5" >
                      <property name="text" >
                        <string>&amp;Composer:</string>
                      </property>
                      <property name="buddy" stdset="0" >
                        <cstring>m_editComposer</cstring>
                      </property>
                      <property name="wordWrap" >
                        <bool>false</bool>
                      </property>
                    </widget>
                  </item>
                  <item row="5" column="0" >
                    <widget class="QLabel" name="textLabel8" >
                      <property name="text" >
                        <string>&amp;Message:</string>
                      </property>
                      <property name="buddy" stdset="0" >
                        <cstring>m_editMessage</cstring>
                      </property>
                      <property name="wordWrap" >
                        <bool>false</bool>
                      </property>
                    </widget>
                  </item>
                  <item row="0" column="1" >
                    <widget class="KLineEdit" name="m_editTitle" />
                  </item>
                  <item row="0" column="2" >
                    <widget class="QLabel" name="textLabel2" >
                      <property name="text" >
                        <string>&amp;Performer:</string>
                      </property>
                      <property name="buddy" stdset="0" >
                        <cstring>m_editPerformer</cstring>
                      </property>
                      <property name="wordWrap" >
                        <bool>false</bool>
                      </property>
                    </widget>
                  </item>
                  <item row="4" column="2" >
                    <widget class="QLabel" name="textLabel6" >
                      <property name="text" >
                        <string>&amp;ISRC:</string>
                      </property>
                      <property name="buddy" stdset="0" >
                        <cstring>m_editIsrc</cstring>
                      </property>
                      <property name="wordWrap" >
                        <bool>false</bool>
                      </property>
                    </widget>
                  </item>
                  <item row="0" column="0" >
                    <widget class="QLabel" name="textLabel1" >
                      <property name="text" >
                        <string>&amp;Title:</string>
                      </property>
                      <property name="buddy" stdset="0" >
                        <cstring>m_editTitle</cstring>
                      </property>
                      <property name="wordWrap" >
                        <bool>false</bool>
                      </property>
                    </widget>
                  </item>
                  <item row="4" column="3" >
                    <widget class="KLineEdit" name="m_editIsrc" />
                  </item>
                  <item row="0" column="3" >
                    <widget class="KLineEdit" name="m_editPerformer" />
                  </item>
                </layout>
              </item>
              <item>
                <spacer name="spacer4" >
                  <property name="sizeHint" >
                    <size>
                      <width>350</width>
                      <height>50</height>
                    </size>
                  </property>
                  <property name="sizeType" >
                    <enum>QSizePolicy::Expanding</enum>
                  </property>
                  <property name="orientation" >
                    <enum>Qt::Vertical</enum>
                  </property>
                </spacer>
              </item>
            </layout>
          </widget>
          <widget class="QWidget" name="tab" >
            <attribute name="title" >
              <string>&amp;Options</string>
            </attribute>
            <layout class="QVBoxLayout" >
              <item>
                <widget class="QCheckBox" name="m_checkPreemphasis" >
                  <property name="text" >
                    <string>Preemph&amp;asis</string>
                  </property>
                  <property name="whatsThis" stdset="0" >
                    <string>&lt;p>Preemphasis is mainly used in audio processing. Higher frequencies in audio signals usually have lower amplitudes. This can lead to bad signal quality on noisy transmission because the high frequencies might become too weak. To avoid this effect, high frequencies are amplified before transmission (preemphasis); the receiver will then weaken them accordingly for playback.</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QCheckBox" name="m_checkCopyPermitted" >
                  <property name="text" >
                    <string>&amp;Copy permitted</string>
                  </property>
                </widget>
              </item>
              <item>
                <layout class="QHBoxLayout" >
                  <item>
                    <widget class="QLabel" name="m_labelPostGap" >
                      <property name="text" >
                        <string>Post-Gap:</string>
                      </property>
                      <property name="wordWrap" >
                        <bool>false</bool>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="K3b::MsfEdit" name="m_editPostGap" >
                      <property name="toolTip" stdset="0" >
                        <string>Set the length of the track's post-gap</string>
                      </property>
                      <property name="whatsThis" stdset="0" >
                        <string>&lt;p>On an audio CD each track (except for the last) can have a post-gap.
This does not mean that K3b adds an additional gap of silence to the track. This setting simply influences the display on a Hifi audio CD player. The part of an audio track that is marked as post-gap is counted backwards.
&lt;p>This setting is irrelevant for most users as modern CD burners can put arbitrary audio data in the post-gap when burning in DAO mode.
&lt;p>&lt;i>In other CD-burning applications the post-gap might be called the pre-gap. The pre-gap of track 2 is the same as the post-gap of track 1.
&lt;p>&lt;b>Changing the post-gap does not change the length of the track.&lt;/b>
&lt;p>&lt;b>When writing in TAO writing mode (not recommended for Audio CDs) the post-gap will most likely be muted and on some burners forced to 2 seconds.&lt;/b></string>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <spacer name="spacer7" >
                      <property name="sizeHint" >
                        <size>
                          <width>161</width>
                          <height>20</height>
                        </size>
                      </property>
                      <property name="sizeType" >
                        <enum>QSizePolicy::Expanding</enum>
                      </property>
                      <property name="orientation" >
                        <enum>Qt::Horizontal</enum>
                      </property>
                    </spacer>
                  </item>
                </layout>
              </item>
              <item>
                <spacer name="spacer3" >
                  <property name="sizeHint" >
                    <size>
                      <width>41</width>
                      <height>80</height>
                    </size>
                  </property>
                  <property name="sizeType" >
                    <enum>QSizePolicy::Expanding</enum>
                  </property>
                  <property name="orientation" >
                    <enum>Qt::Vertical</enum>
                  </property>
                </spacer>
              </item>
            </layout>
          </widget>
        </widget>
      </item>
    </layout>
  </widget>
  <customwidgets>
    <customwidget>
      <class>K3b::MsfEdit</class>
      <extends>QWidget</extends>
      <header location="global" >k3bmsfedit.h</header>
      <sizehint>
        <width>100</width>
        <height>-1</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>7</hordata>
        <verdata>5</verdata>
      </sizepolicy>
      <pixmap>image0</pixmap>
    </customwidget>
  </customwidgets>
  <tabstops>
    <tabstop>m_editTitle</tabstop>
    <tabstop>m_editPerformer</tabstop>
    <tabstop>m_editArranger</tabstop>
    <tabstop>m_editComposer</tabstop>
    <tabstop>m_editSongwriter</tabstop>
    <tabstop>m_editIsrc</tabstop>
    <tabstop>m_editMessage</tabstop>
    <tabstop>m_checkPreemphasis</tabstop>
    <tabstop>m_checkCopyPermitted</tabstop>
  </tabstops>
  <images>
    <image name="image0" >
      <data format="XBM.GZ" length="79" >789c534e494dcbcc4b554829cdcdad8c2fcf4c29c95030e0524611cd48cd4ccf28010a1797249664262b2467241641a592324b8aa363156c15aab914146aadb90067111b1f</data>
    </image>
  </images>
  <includes>
    <include location="local" >klineedit.h</include>
    <include location="local" >klineedit.h</include>
    <include location="local" >klineedit.h</include>
    <include location="local" >klineedit.h</include>
    <include location="local" >klineedit.h</include>
    <include location="local" >klineedit.h</include>
    <include location="local" >klineedit.h</include>
    <include location="local" >k3bmsfedit.h</include>
  </includes>
</ui>
